scripts/update-version.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+if [ $# -eq 0 ]; then
+    echo "Usage: v<semantic version>"
+    exit 1
+fi
+
+SEM_VERSION=$1
+SEM_VERSION=${SEM_VERSION/#v}
+SHORT_VERSION=${SEM_VERSION%-*}
+MINOR_VERSION=${SEM_VERSION%.*}
+YEAR=$(date '+%Y')
+
+step "Version ${SEM_VERSION}"
+
+step "Updating Xcode targets to version ${SHORT_VERSION}…"
+
+xcrun agvtool bump -all
+xcrun agvtool new-marketing-version "${SHORT_VERSION}"
+
+step "Updating CocoaPods podspecs to version ${SEM_VERSION}…"
+
+find . -type f -name '*.podspec' -exec sed -i '' "s/^ *s.version *=.*$/  s.version = \"${SEM_VERSION}\"/" {} +
+
+#step "Updating changelog to version ${SHORT_VERSION}…"
+#
+#sed -i '' -E "s/## *master/## ${SHORT_VERSION}/g" CHANGELOG.md
+
+# Skip updating the installation instructions for patch releases or prereleases.
+if [[ $SHORT_VERSION == $SEM_VERSION && $SHORT_VERSION == *.0 ]]; then
+    step "Updating readmes to version ${SEM_VERSION}…"
+    sed -i '' -E "s/~> *[^']+/~> ${MINOR_VERSION}/g; s/from: \"*[^\"]+/from: \"${SEM_VERSION}/g" README.md
```

Comment:
nit: `readme.md` is actually lowercased. Let's `git mv readme.md README.md` for consistency and if we ever want to run this on a case-sensitive file system.